packages/kbn-typed-react-router-config/src/create_router.ts

```diff
@@ -25,22 +25,24 @@ import { Route, Router } from './types';
 const deepExactRt: typeof deepExactRtTyped = deepExactRtNonTyped;
 const mergeRt: typeof mergeRtTyped = mergeRtNonTyped;
 
+function toReactRouterPath(path: string) {
+  return path.replace(/(?:{([^\/]+)})/, ':$1');
+}
+
 export function createRouter<TRoutes extends Route[]>(routes: TRoutes): Router<TRoutes> {
   const routesByReactRouterConfig = new Map<ReactRouterConfig, Route>();
   const reactRouterConfigsByRoute = new Map<Route, ReactRouterConfig>();
 
   const reactRouterConfigs = routes.map((route) => toReactRouterConfigRoute(route));
 
-  function toReactRouterConfigRoute(route: Route, prefix: string = ''): ReactRouterConfig {
-    const path = `${prefix}${route.path}`.replace(/\/{2,}/g, '/').replace(/\/$/, '') || '/';
+  function toReactRouterConfigRoute(route: Route): ReactRouterConfig {
```

Comment:
Should we have a unit test for `toReactRouterConfigRoute` or `toReactRouterPath` to document this conversion?

Reply:
I've tried to stick to testing the actual API of the package rather than implementation details. I think it's worked out well so far, would you be OK with a comment?